src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,64 @@
+$square-size: 100px;
+$interval-size: 1px;
+
+@for $dayNumber from 1 through 31 {
+  .calendar__square-#{$dayNumber}::before {
+    content: "#{$dayNumber}";
+    font-size: 30px;
+    font-family: Arial, Helvetica, sans-serif;
+    margin: auto;
+  }
+
+  .calendar__square-#{$dayNumber} {
+    width: $square-size;
+    height: $square-size;
+    position: relative;
+    background-color: #eee;
+    border: 1px solid black;
+    box-sizing: border-box;
+    display: flex;
+    cursor: pointer;
+  }
+
+  .calendar__square-#{$dayNumber}:hover {
+    background-color: pink;
+    transition-duration: 0.5s;
+    transform: translate(0, -20px);
+  }
```

Comment:
```suggestion
    box-sizing: border-box;
    display: flex;
    cursor: pointer;
    transition: background-color 0.5s, transform 0.5s;
  }

  .calendar__square-#{$dayNumber}:hover {
    background-color: pink;
    transform: translate(0, -20px);
  }
```

https://user-images.githubusercontent.com/94289038/176638088-5911046e-759b-4358-b5fa-9e3b21bcf55e.mov

transition should be smooth in both directions